content/messaging/customer-chat-api/changelog/index.mdx

```diff
@@ -15,6 +15,7 @@ This document is the record of all the changes in the **Customer Chat API**, Web
 
 - There are new methods: **Get Customer** ([Web](/messaging/customer-chat-api/v3.2/#get-customer) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#get-customer)), **Accept Greeting** ([Web](/messaging/customer-chat-api/v3.2/#accept-greeting) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#accept-greeting)), **Cancel Greeting** ([Web](/messaging/customer-chat-api/v3.2/#cancel-greeting) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#cancel-greeting)), **List License Properties** ([Web](/messaging/customer-chat-api/v3.2/#list-license-properties)), **List Group Properties** ([Web](/messaging/customer-chat-api/v3.2/#list-group-properties)), **List Threads** ([Web](/messaging/customer-chat-api/v3.2/#list-threads) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#list-threads)).
 - There are new pushes: [`incoming_greeting`](/messaging/customer-chat-api/v3.2/rtm-reference/#incoming_greeting), [`greeting_accepted`](/messaging/customer-chat-api/v3.2/rtm-reference/#greeting_accepted), and [`greeting_canceled`](/messaging/customer-chat-api/v3.2/rtm-reference/#greeting_canceled).
+- Added new parameter for **Greeting**
```

Comment:
```suggestion
- Added new `addon` parameter for **Greeting**
```

Also, was there `greeting` object in v3.1? If not then I guess the information on line 28 is sufficient.

Reply:
But we also adding new property to `incomming_greeting` push which was available earlier.